ballerina-to-openapi/src/main/java/io/ballerina/openapi/service/mapper/OpenAPIParameterMapper.java

```diff
@@ -68,34 +67,39 @@ public class OpenAPIParameterMapper {
     private final FunctionDefinitionNode functionDefinitionNode;
     private final OperationAdaptor operationAdaptor;
     private final Map<String, String> apidocs;
-    private final List<OpenAPIMapperDiagnostic> errors;
+    private final List<OpenAPIMapperDiagnostic> diagnostics = new ArrayList<>();
     private final SemanticModel semanticModel;
     private final ModuleMemberVisitor moduleMemberVisitor;
     private final ComponentMapper componentMapper;
+    private final OpenAPI openAPI;
+
+    public List<OpenAPIMapperDiagnostic> getErrors() {
```

Comment:
Usually the first method in a Java class is the constructor. Hence, shall we move this method into an appropriate place in the file ?